dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/resource/DigitalObjectRenderingFileResourceRepositoryImpl.java

```diff
@@ -0,0 +1,104 @@
+package de.digitalcollections.cudami.server.backend.impl.jdbi.identifiable.resource;
+
+import de.digitalcollections.cudami.model.config.CudamiConfig;
+import de.digitalcollections.cudami.server.backend.api.repository.identifiable.resource.DigitalObjectRenderingFileResourceRepository;
+import de.digitalcollections.model.identifiable.resource.FileResource;
+import de.digitalcollections.model.identifiable.resource.FileResourceType;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.core.statement.PreparedBatch;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class DigitalObjectRenderingFileResourceRepositoryImpl
+    extends FileResourceMetadataRepositoryImpl<FileResource>
+    implements DigitalObjectRenderingFileResourceRepository {
+
+  @Autowired
+  public DigitalObjectRenderingFileResourceRepositoryImpl(Jdbi dbi, CudamiConfig cudamiConfig) {
+    super(dbi, cudamiConfig);
+  }
+
+  @Override
+  public List<FileResource> getForDigitalObjectUuid(UUID digitalObjectUuid) {
+    final String rfrTableAlias = getTableAlias();
+    final String rfrTableName = getTableName();
+    final String fieldsSql = getSqlSelectAllFields();
+    StringBuilder innerQuery =
+        new StringBuilder(
+            "SELECT dr.sortindex as idx, * FROM "
+                + rfrTableName
+                + " AS "
+                + rfrTableAlias
+                + " INNER JOIN digitalobject_renderingresources AS dr ON "
+                + rfrTableAlias
+                + ".uuid = dr.fileresource_uuid"
+                + " WHERE dr.digitalobject_uuid = :uuid"
+                + " ORDER by idx ASC");
```

Comment:
Bitte generell bei Order-bys den qualifizierten Tabellennamen nutzen! Das ist v.a. fürs Paging unerlässlich. Wir sollten das daher überall so machen.
```suggestion
                + " ORDER by dr.sortindex ASC");
```